test.py

```diff
@@ -0,0 +1,19 @@
+import dialogue as d
+
+if __name__ == '__main__':
```

Comment:
This should live in `tests`, optionally we can use `unittest` to test that good runs do good things.